# Generated by Django 3.1.13 on 2021-09-03 14:09

from django.db import migrations, models

import tapir.accounts.models
import tapir.accounts.validators


class Migration(migrations.Migration):

    dependencies = [
        ("accounts", "0006_use_phone_number_field"),
    ]

    operations = [
        migrations.AlterModelManagers(
            name="tapiruser",
            managers=[
                ("objects", tapir.accounts.models.TapirUserManager()),
            ],
        ),
        migrations.AlterField(
            model_name="tapiruser",
            name="username",
            field=models.CharField(
                error_messages={"unique": "A user with that username already exists."},
                help_text="Required. 150 characters or fewer. Letters, digits and ./-/_ only.",
                max_length=150,
                unique=True,
                validators=[tapir.accounts.validators.UsernameValidator()],
                verbose_name="username",
            ),
        ),
    ]
